sdk/storage/Azure.Storage.Common/tests/Shared/StorageTestBase.cs

```diff
@@ -470,5 +471,20 @@ protected async Task<Stream> CreateLimitedMemoryStream(long size, long maxMemory
             }
             return stream;
         }
+
+#pragma warning disable SA1402 // File may only contain a single type
```

Comment:
Style nit - We generally move these to the top of the file instead of wrapping each new class.